go.mod

```diff
@@ -24,5 +24,5 @@ require (
 // For local override of dependencies, use following:
 // replace github.com/rh-messaging/activemq-artemis-operator v0.0.0+incompatible => ../../../github.com/rh-messaging/activemq-artemis-operator
 
-//replace github.com/rh-messaging/shipshape v0.0.0 => ../../../github.com/rh-messaging/shipshape
+replace github.com/rh-messaging/shipshape v0.0.0 => ../../../github.com/rh-messaging/shipshape
```

Comment:
I guess that you forgot to revert the ss dependency from you local to a specific version.